voorhoede/head-start
# chore: upgrade to Astro v4

# Changes

- upgrade to Astro v4
- upgrade Astro related packages
- fix issues caused by upgrades

Easy peasy with the new `npx @astrojs/upgrade`:

![image](https://github.com/voorhoede/head-start/assets/1516059/6660090b-1100-40b6-8df3-b310e691cff8)

# Associated issue

N/A

# How to test

1. Open preview link
2. Verify everything still works as it did before

# Checklist

- ~~I have added/updated tests to prove that my feature works (if not possible please explain why)~~
- ~~I have made changes to the README and if the change affects the project setup (npm commands changed, new service added, environmental variable added)~~
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
